### PR TITLE
f-development-context@1.2.0 - new log interface signature

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- $logger interface signature to $log.

### CODE REVIEW

**Review by oliversweb — DISMISSED** [review]

File: packages/services/f-development-context/src/logger.context.js [hunk 1]
```diff
@@ -7,17 +7,17 @@ const appendStandardProperties = (logPayload, message, level, mode) => {
...
     },
-    logWarn: (logMessage, store, logPayload = {}) => {
+    earn: (logMessage, tags, logPayload = {}) => {
         appendStandardProperties(logPayload, logMessage, 'Warn', 'client');
-        console.warn(logMessage, logPayload);
+        console.warn(logMessage, tags, logPayload);
     },
-    logInfo: (logMessage, store, logPayload = {}) => {
+    enfo: (logMessage, tags, logPayload = {}) => {
         appendStandardProperties(logPayload, logMessage, 'Info', 'client');
```
Comment (AlixRichards) [inline packages/services/f-development-context/src/logger.context.js:19]: Did you want these to be `warn` and `info`?

File: packages/storybook/context/logger.context.js [hunk 2]
```diff
@@ -8,17 +8,17 @@ export default () => {
     };
 
     const logger = {
-        logError: (logMessage, store, logPayload = {}) => {
+        error: (logMessage, exception, tags, logPayload = {}) => {
             appendStandardProperties(logPayload, logMessage, 'Error', 'client');
```
Comment (AlixRichards) [inline packages/storybook/context/logger.context.js:12]: is it ok that this is different to the one above?

Above: `error: (logMessage, error, tags, logPayload = {}) => {`
Here: `error: (logMessage, exception, tags, logPayload = {}) => {`

File: packages/services/f-development-context/src/logger.context.js (hunk 1 above)
Comment (dandel10n) [reply packages/services/f-development-context/src/logger.context.js:19]: Oh my god :ashamed:

File: packages/storybook/context/logger.context.js (hunk 2 above)
Comment (dandel10n) [reply packages/storybook/context/logger.context.js:12]: Good shout, I changed everything to exception, thanks!

**Review by AlixRichards — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**